sync/_databricks.py

```diff
@@ -296,9 +298,9 @@ def create_submission_for_run(
 
     if project_id in project_cluster_tasks:
         cluster_id, tasks = project_cluster_tasks.get(project_id)
-    elif None in project_cluster_tasks and len(project_cluster_tasks) == 1:
-        # If there's only 1 cluster and it's not tagged with project ID assume that's the one for the project
-        cluster_id, tasks = project_cluster_tasks.get(None)
+    elif len(project_cluster_tasks) == 1:
+        # If there's only 1 cluster then assume that's the one for the project
+        cluster_id, tasks = next(iter(project_cluster_tasks.values()))
```

Comment:
I loosened this restriction because it was preventing me from "resubmitting" existing job runs to a new project-id. This is a super handy capability for development when we want to retest the submission/rec cycle with running new dbx jobs.

Reply:
I'll allow this ;-)